File: ServiceBook/ClassLibrary1/Models/VehicleServiceCompanyOwner.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServiceBook.DAL.Models
{
    public class VehicleServiceCompanyOwner
    {
        [Key]
        public int ID { get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }
        public virtual ICollection<VehicleServiceCompany> VehicleServiceCompanys { get; set; }
    }
}

File: ServiceBook/ServiceBook/Scripts/Service/Models/VehicleServiceCompanyOwner.js
    function VehicleServiceCompanyOwner(data)
    {
        this.id=null;
        this.phoneNumber=null;
        this.firstName=null;
        this.lastName=null;
        this.email=null;
        this.vehicles=null;
        this.uid = null;
        this.serviceCompanies = ko.observableArray(null);
        if(data!=null)
        {
            this.id=data.ID;
            this.phoneNumber=data.PhoneNumber;
            this.firstName=data.FirstName;
            this.lastName=data.LastName;
            this.email=data.email;
            this.uid=data.UID;
            var serviceCompanies = _.map(data.VehicleServiceCompanies, function (vehicleServieCompany, index) {
                return new VehicleServiceCompany(vehicleServieCompany);

            });
           
        }
    }